solidity-frontend/src/main/java/org/sonarsource/solidity/frontend/Utils.java

```diff
@@ -34,4 +35,15 @@ public static SolidityParser returnParserFromParsedFile(CharStream cs) {
     parser.handleComments(tokenStream.getTokens());
     return parser;
   }
+
+  public static void handleStructuredComments(Token token) {
+    // System.out.println(token);
+    char[] temp = token.getText().toCharArray();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "temp" local variable. ([squid:S1481](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS1481))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=15&issues=AWQDQR7h3e-qcckjGvIk&open=AWQDQR7h3e-qcckjGvIk)

[ISSUE_KEY]: AWQDQR7h3e-qcckjGvIk